yt_dlp/extractor/gofile.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    try_get
+)
+
+
+class GofileIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?gofile\.io/d/(?P<id>[^/]+)'
+    _TESTS = [{
+        'url': 'https://gofile.io/d/AMZyDw',
+        'md5': '774879ad31bc62c23269da8bf490d025',
+        'info_dict': {
+            'id': 'de571ac1-5edc-42e2-8ec2-bdac83ad4a31',
+            'filesize': 928116,
+            'ext': 'mp4',
+            'title': 'nuuh'
+        }
+    }]
+
+    def _entries(self, file_id):
+
+        # Create guest account
+        accountdata = self._download_json_handle('https://api.gofile.io/createAccount', 'Gofile', note='Getting a new guest account')
+        accountToken = accountdata[0]['data']['token']
```

Comment:
Try to see if u can cache this token so the request is not made for every url

Reply:
entries() is only ran once, so token should only be generated once. Do you mean cache token between different runs of the program?